monero-outreach-docs/translations/pt-br/quickfacts/QuickFacts_pt-br.md

```diff
@@ -0,0 +1,125 @@
+
+**A mineração é acessível:** Qualquer um com um dispositivo ou navegador conectado por participar.
+
+### IMPLICAÇÕES E USOS NO MUNDO REAL
+
+Pelo Monero ser seguro, com baixas taxas e sem fronteiras, as pessoas podem facilmente enviar dinheiro independentemente de governos ou bancos corrúptos e falidos. Da mesma forma, negócios podem ser administrados sem competidores bisbilhotando informações críticas. Isso permite o empoderamento econômico dos indivíduos e dos negócios em **[países opressores](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)**, em depressão econômica ou em ambientes de negócios altamente competitivos.
+
+Um histórico financeiro privado protege consumidores e companhias de manipulação de preços, abuso da cadeia de suprimentos, discriminação econômica, entre outras coisas. **[Monero é a única criptomoeda](https://www.reddit.com/r/Monero/comments/8k8pk9/monero_the_worlds_bestkept_secret/)** que tem os recursos necessários para funcionar como dinheiro digital completamente fungível e descentralizado.
+
+### FUNDAMENTOS TÉCNICOS
+
+_(Até 1/2/2019)_
++ **Quantidade de nós ativos:** 1,564 (Fonte: https://monerohash.com/nodes-distribution.html)
++ **Taxa de hash da rede:** 663.2 MH/s
++ **Média de transações/hora:** 149 (média 30 dias)
++ **Núcleos CPU protegendo a rede:** 22,234,297
++ **Monero em circulação:** 16,758,217 XMR (Aproximadamente)
++ **Capitalização de mercado:** $732,932,071 USD (~0.65% da cap. total do mercado de criptomoedas)
++ **Recompensa do bloco atualmente:** 3.22 XMR
++ **Intervalo médio entre blocos:** 2 Minutos
+
+_A taxa de recompensas diminuirá continuamente até o final de Maio de 2022, quando haverão 18,132 milhões de XMR em circulação. Neste ponto uma recompensa de 0,6 XMR por bloco se manterá indefinidamente._
+
+_Com uma "emissão em cauda" de 0,6 XMR/bloco, em 2040 haverá uma quantidade igual de Monero e Bitcoin (aprox. 21 milhões)_
+
+### RECURSOS EM DESENVOLVIMENTO
+
+Embora o Monero já esteja disponível e em uso através do globo, a comunidade de desenvolvedores tem objetivos excitantes para continuar melhorando a privacidade, a segurança e a usabilidade do Monero e das criptomoedas em geral. Estas são algumas das coisas que estão por vir:
+
+**Kovri:** Esta é uma das maiores melhorias, adicionando uma quarta camada de privacidade às transações do Monero. Kovri utiliza **[criptografia garlic](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (pense em Tor com esteroídes) para esconder o endereço IP dos remetentes e destinatários usando a rede. Cada camada de uma transação Monero, do remetente, destinatário, quantidade transacionada, e (com Kovri) o pilar central da transação em si, serão privados e seguros.

```

Comment:
- _criptografia garlic_/**Criptografia em Alho**: https://www.getmonero.org/pt-br/resources/moneropedia/garlic-encryption.html

On getmonero.org _"garlic encryption"_ is translated as **"Criptografia em Alho"**. It would be nice to verify if this is correct.